client/src/routes/stats/+page.svelte

```diff
@@ -17,13 +18,18 @@
   let catfishLosses: number;
   let catRatio: number;
   let catfishRatio: number;
+  let userId: string | null;
+
+  onMount(() => {
+    userId = $page.url.searchParams.get("user");
```

Comment:
you can remove the declaration outisde the onMount and make it a const inside onMount